src/raylib.h

```diff
@@ -1492,8 +1492,10 @@ RLAPI void ResumeMusicStream(Music music);                            // Resume
 RLAPI bool IsMusicPlaying(Music music);                               // Check if music is playing
 RLAPI void SetMusicVolume(Music music, float volume);                 // Set volume for music (1.0 is max level)
 RLAPI void SetMusicPitch(Music music, float pitch);                   // Set pitch for a music (1.0 is base level)
+RLAPI void SetMusicLooping(Music *music, bool loop);                   // Set the music to loop or not
```

Comment:
As commented above, I prefer to avoid those functions and allow the direct access to `music.looping`, like other data structures in raylib.